Userland/Libraries/LibWeb/Animations/KeyframeEffect.cpp

```diff
@@ -871,6 +871,59 @@ void KeyframeEffect::visit_edges(Cell::Visitor& visitor)
         visitor.visit(keyframe);
 }
 
+struct RequiredInvalidationAfterStyleChange {
+    bool repaint { false };
+    bool rebuild_stacking_context_tree { false };
+    bool relayout { false };
+    bool rebuild_layout_tree { false };
+
+    static RequiredInvalidationAfterStyleChange full() { return { true, true, true, true }; }
+};
```

Comment:
This struct already exists in Element! Let's not duplicate it.

Reply:
done, moved it into separate header.